pkg/upload/react-native.go

```diff
@@ -0,0 +1,76 @@
+package upload
+
+import (
+	"path/filepath"
+
+	"github.com/bugsnag/bugsnag-cli/pkg/log"
+	"github.com/bugsnag/bugsnag-cli/pkg/options"
+	"github.com/bugsnag/bugsnag-cli/pkg/utils"
+)
+
+func ProcessReactNative(globalOptions options.CLI, endpoint string, logger log.Logger) error {
+	reactNativeOptions := globalOptions.Upload.ReactNative
+
+	// Some commands must be run from either the android/ or ios/ subdirectory.
```

Comment:
I believe the two source map commands can be run from inside their respective platform directories, so perhaps use these paths in each command for consistency?